lib/gollum-lib/page.rb

```diff
@@ -281,6 +281,14 @@ def versions(options = {})
       @wiki.repo.git.versions_for_path(@path, @wiki.ref, log_pagination_options(options))
     end
 
+    # Public: The last version that has touched the Page. Can be nil.
+    #
+    # Returns Gollum::Git::Commit, or nil.
+    def last_version
+      return @last_version if defined? @last_version
```

Comment:
This is the intended behavior, because we want to cache a possible `nil` value for `@last_version` (it will be `nil` if the Page is a preview).